lib/active_model/serializer/adapter/fragment_cache.rb

```diff
@@ -51,20 +51,23 @@ def cached_attributes(klass, serializers)
           non_cached_attributes = attributes - cached_attributes
 
           cached_attributes.each do |attribute|
-            options = serializer.class._attributes_keys[attribute]
-            options ||= {}
-            # Add cached attributes to cached Serializer
-            serializers[:cached].constantize.attribute(attribute, options)
+            add_attribute_to_serializer(attribute, serializers, :cached)
           end
 
           non_cached_attributes.each do |attribute|
-            options = serializer.class._attributes_keys[attribute]
-            options ||= {}
-            # Add non-cached attributes to non-cached Serializer
-            serializers[:non_cached].constantize.attribute(attribute, options)
+            add_attribute_to_serializer(attribute, serializers, :non_cached)
           end
         end
 
+        # Given an attribute and the key of its serializer
+        # 1. Add attribute to the appropriate cached or non-cached Serializer
+        def add_attribute_to_serializer(attribute, serializers, key)
```

Comment:
Sorry. Busy few weeks. I'll try to tackle this tonight or this weekend at the latest. 

> On Dec 8, 2015, at 10:24 PM, Benjamin Fleischer <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/adapter/fragment_cache.rb:
> 
> >            end
> >          end
> >  
> > +        # Given an attribute and the key of its serializer
> > +        # 1. Add attribute to the appropriate cached or non-cached Serializer
> > +        def add_attribute_to_serializer(attribute, serializers, key)
> bump?
> 
> —
> Reply to this email directly or view it on GitHub.
>